webpush/encrypt.go

```diff
@@ -139,36 +142,64 @@ type EncryptionResult struct {
 	ServerPublicKey []byte
 }
 
+// Decrypt an encrypted messages.
```

Comment:
I *tend* to agree that this package could use a `Decrypt()` function. However, if there’s any concerns about API contracts or maintainability, I think covering the 99% use-case of sending messages only. 

Your call, @wibblymat.